File: assets/Practice/practice01/skeleton/sum_integers.c
// sum_integers.c
// This program computes sum of positive integers up to n
#include <stdio.h>

int main(void) {
	int n; // upper limit, a positive integer
	int count=1, ans=0; // initialisation

	printf("Enter n: ");
	scanf("%d", &n);

	while (count <= n) {
		ans += count;
		count++;
	}
	printf("Sum = %d\n", ans);

	return 0;
}

File: assets/Practice/practice01/skeleton/sum_integers.py
# sum_integers.py
# This program computes sum of positive integers up to n

import sys

# main function
def main():
	n = int(raw_input("Enter n: "))
	count = 1
	ans = 0

	while count <= n:
		ans += count
		count += 1

	print "Sum = %d" % ans

# Runs the main method
if __name__ == "__main__":
	main()
	sys.exit(0)
